third_party/Android.mk

```diff
@@ -89,15 +89,23 @@ SPVTOOLS_OUT_PATH=$(abspath $(TARGET_OUT))
 
 define gen_spvtools_grammar_tables
 $(call generate-file-dir,$(SPVTOOLS_OUT_PATH)/opcode.inc)
-$(1)/opcode.inc $(1)/operand.inc: $(SPVTOOLS_LOCAL_PATH)/utils/generate_grammar_tables.py \
-                                  $(SPVTOOLS_LOCAL_PATH)/source/grammar.json
+$(1)/core.insts.inc $(1)/operand.kinds.inc $(1)/glsl.std.450.insts.inc $(1)/opencl.std.insts.inc: \
+        $(SPVTOOLS_LOCAL_PATH)/utils/generate_grammar_tables.py \
+        $(SPVTOOLS_LOCAL_PATH)/source/spirv.core.grammar.json \
+        $(SPVTOOLS_LOCAL_PATH)/source/extinst.glsl.std.450.grammar.json \
+        $(SPVTOOLS_LOCAL_PATH)/source/extinst.opencl.std.grammar.json
 		@$(HOST_PYTHON) $(SPVTOOLS_LOCAL_PATH)/utils/generate_grammar_tables.py \
-		                $(SPVTOOLS_LOCAL_PATH)/source/grammar.json \
-		                --opcode-file=$(1)/opcode.inc \
-		                --operand-file=$(1)/operand.inc
-		@echo "[$(TARGET_ARCH_ABI)] Grammar        : opcode.inc & operand.inc <= grammar.json"
-$(SPVTOOLS_LOCAL_PATH)/source/opcode.cpp: $(1)/opcode.inc
-$(SPVTOOLS_LOCAL_PATH)/source/operand.cpp: $(1)/operand.inc
+		                --spirv-core-grammar=$(SPVTOOLS_LOCAL_PATH)/source/spirv.core.grammar.json \
+		                --extinst-glsl-grammar=$(SPVTOOLS_LOCAL_PATH)/source/extinst.glsl.std.450.grammar.json \
+		                --extinst-opencl-grammar=$(SPVTOOLS_LOCAL_PATH)/source/extinst.opencl.std.grammar.json \
+		                --core-insts-output=$(1)/core.insts.inc \
+		                --glsl-insts-output=$(1)/glsl.std.450.insts.inc \
```

Comment:
That's fair, I don't have a strong opinion, it just seems a bit unfortunate to have a single makefile rule that generates 4 different outputs. (And it's fairly conceivable to see it grow in the future)